src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -271,13 +269,4 @@ const ArticlePage = ({ pageData }) => {
   );
 };
 
-ArticlePage.propTypes = {
-  pageData: articleDataPropTypes.isRequired,
-  mostReadEndpointOverride: string,
-};
-
-ArticlePage.defaultProps = {
-  mostReadEndpointOverride: null,
```

Comment:
Does this need adding above?

Reply:
Can't see this being used anywhere in the component, so I removed it entirely.